src/js/components/modals/AppModalComponent.jsx

```diff
@@ -235,6 +240,10 @@ var AppModalComponent = React.createClass({
           size="md"
           onDestroy={this.props.onDestroy}>
         <form method="post" role="form" onSubmit={this.handleSubmit}>
+          <button onClick={event => event.preventDefault()}
+              className="enter-trap">
+            Enter trap
+          </button>
```

Comment:
I wanted it to be very descriptive so that everybody does know what it is for, But actually you are right. It is over descriptive.